eslint-local-rules/secureCommandExecution.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Check command execution within nodejs scripts',
+    },
+    schema: [],
+  },
+  create(context) {
+    return {
+      'TaggedTemplateExpression[tag.name="command"]'(node) {
+        if (!isCommandIsExecuted(node)) {
+          context.report({
+            node,
+            message: 'Command is missing a `run()` call',
+          })
+        }
+
+        if (isCommandContainsShellCharacters(node)) {
+          context.report({
+            node,
+            message: "Command is containing shell characters. This is probably a mistake as it won't run in a shell.",
+          })
+        }
+      },
+    }
+  },
+}
+
+function isCommandIsExecuted(node) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
function isCommandExecuted(node) {
```